cmd/object-handlers.go

```diff
@@ -534,6 +534,24 @@ func (api objectAPIHandlers) HeadObjectHandler(w http.ResponseWriter, r *http.Re
 			case crypto.S3.IsEncrypted(objInfo.UserDefined):
 				w.Header().Set(crypto.SSEHeader, crypto.SSEAlgorithmAES256)
 			case crypto.SSEC.IsEncrypted(objInfo.UserDefined):
+				sealedKey, err := crypto.SSEC.ParseMetadata(objInfo.UserDefined)
```

Comment:
We need a helper function here for objInfo @kannappanr we don't need to pile all this here move this to cmd/encryption-v1.go

Reply:
done

Reply:
@kannappanr We can use: #6609